src/platform_impl/windows/accelerator.rs

```diff
@@ -7,19 +7,20 @@ use std::{
 };
 
 use lazy_static::lazy_static;
-use winapi::{ctypes::c_int, shared::windef::*, um::winuser::*};
+
+use webview2_com_sys::Windows::Win32::{Foundation::HWND, UI::WindowsAndMessaging::*};
 
 // NOTE:
 // https://docs.microsoft.com/en-us/windows/win32/wsw/thread-safety
 // All handles you obtain from functions in Kernel32 are thread-safe,
 // unless the MSDN Library article for the function explicitly mentions it is not.
 
 #[derive(Debug, Copy, Clone, Eq, PartialEq, Hash)]
-struct WindowHandle(HWND);
+struct WindowHandle(isize);
```

Comment:
If I recall correctly, it's because `HWND` and several other tuple types don't implement `Send`, but the simple handle values inside can be passed. I should go double check why they don't implement that trait, but I don't think it's anything like implementing `Drop` (which would be bad if the internal value were stolen).

BTW, something like this is what I had in mind for the types in the public interface. Strongly typed, passed around by value, but opaque to consumers.

Reply:
I see, then wrapping `isize` here is fine.